notebooks/tel_and_site/subsys_req_ver/m2/SITCOMTN-096_M2_outer_control_closed_loop.ipynb

```diff
@@ -0,0 +1,1318 @@
+{
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #5.    for i in [i for i in range(total_tangent_number) if i % 2 == 0]:
</pre><p>To make the code more readable, put the condition on the index inside the for loop like</p><p> </p><pre class="ql-syntax" spellcheck="false">for i in range(totoal_tangent_number):
	if i % 2 != 0: continue
</pre>
<br/>

 _Reply via <a href='https://app.reviewnb.com/lsst-sitcom/notebooks_vandv/pull/85/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='25'/>